lib/node_modules/@stdlib/assert/is-ragged-nested-array/examples/index.js

```diff
@@ -0,0 +1,50 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+/* eslint-disable no-empty-function, no-restricted-syntax */
+
+'use strict';
+
+var isRaggedNestedArray = require( './../lib' );
+
+console.log( isRaggedNestedArray( [ [1, 2, 3], [4, 5] ] ) );
+// => true
+
+console.log( isRaggedNestedArray( [ [1, 2, 3], [4, 5, 6] ] ) );
```

Comment:
```suggestion
console.log( isRaggedNestedArray( [ [ 1, 2, 3 ], [ 4, 5 ] ] ) );
// => true

console.log( isRaggedNestedArray( [ [ 1, 2, 3 ], [ 4, 5, 6 ] ] ) );
```

Here and elsewhere.

Reply:
Thanks for the suggestion @kgryte! Just to confirm, you mean adding a leading and trailing whitespace in array initializations in all files in the package not just this one right?